gromacs/core.py

```diff
@@ -600,44 +600,37 @@ def transform_args(self,*args,**kwargs):
         return self._build_arg_list(**newargs)
 
     def _get_gmx_docs(self):
-        """Extract standard gromacs doc by running the program and chopping the header.
+        """Extract standard gromacs doc
 
-        .. Note::
-
-           The header is on STDOUT and is ignored. The docs are read from STDERR in GMX 4.
```

Comment:
Leave the note in... it took a while to figure it out and it is valuable to keep this information around. This doc string is not even shown to users so it really is not a problem because it won't confuse them. We could make it a code comment. But in any case, leave it.